dephell/networking.py

```diff
@@ -3,8 +3,14 @@
 
 # external
 import certifi
+import requests
 from aiohttp import ClientSession, TCPConnector
 
+# app
+from . import __version__
+
+USER_AGENT = {'User-Agent': 'DepHell/{version}'.format(version=__version__)}
```

Comment:
This must be a string, not a dict.

Reply:
Because you're passing it as a value in headers dict.

Reply:
Ah yes, updated!